lectureDL.py

```diff
@@ -152,26 +167,30 @@ def check_uni_folder(uni_folder, home_dir):
     @param: uni_folder - pathname generated using os.path
     '''
     if not os.path.exists(uni_folder):
-        conf = input(f"{uni_folder}{FOLDER_ERROR}")[0].lower()
+        conf = input(f"{uni_folder}{FOLDER_ERROR}")[FIRST_CHAR].lower()
         if conf != 'y':
             print('Ok, shutting down.')
             sys.exit(1)
         uni_folder = os.path.join(home_dir, "Downloads")
     return uni_folder
 
 
-def getSubjectFolder(fname, uni_folder):
-    subjectCode = fname.split()[0].lower()
-    print("Getting subject folder")
-    print(f"Fname: {fname}, subjectCode: {subjectCode}")
+def getSubjectFolder(subject_code, uni_folder):
+    ''' Enables any subject_code in which the subject code is included to be
+    identified as the appropriate folder for the subject.
+    '''
+    print(f"Retrieving folder with name that includes: {subject_code}")
     # Using the subject code to find the appropriate folder.
-    for i in os.listdir(uni_folder):
-        if subjectCode.lower() in i.lower():
-            subjectFolder = i
+    for fold in os.listdir(uni_folder):
+        if subject_code.lower() in fold.lower():
+            subjectFolder = fold
             break
     try:
         return subjectFolder
     except NameError as e:
+        print(f"No folder including the code '{subject_code}' found.",
+              "Either create such a folder manually, or retry with",
+              "'auto_create_subfolders' setting set to True.")
```

Comment:
This will change your life, but you don't actually need the commas. If you put two strings beside each other in Python 3, they are just automatically concatenated, even across lines.

E.g.
```
>>> print(f"hey "
...        "friend")
hey friend
```

Reply:
Haha amazing, thanks!